File: Backup11-04-2021/src/app/model/questionpool.ts
import { User } from './user';
export class Questionpool {
    questionpoolId: Int32Array;
    questionpoolName: string;
    createdDate: any;
    lastEdited: any;
    hastag:string;
    thumbnailImage: string;
    thumbnailImageURL: string;
    isActive:boolean;
    //quizCode: string;
    courseId: Int32Array;
    lessonId: number;
    quizCount:number;
    dateDistance:any;
    user : User;
}

File: Backup07-05-2021/API/API/API/Models/Questionpool.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace API.Models
{
  public class Questionpool
  {
    [Key]
    public int QuestionpoolId { get; set; }
    public string QuestionpoolName { get; set; }
    //public string Question { get; set; }
    //public int Point { get; set; }
    //public int Time { get; set; }
    public DateTime CreatedDate { get; set; }
    public DateTime LastEdited { get; set; }
    public string Hastag { get; set; }
    public Byte[] ThumbnailImage { get; set; }
    public string ThumbnailImageURL { get; set; }
    public Boolean IsActive { get; set; }
    //public string QuizCode { get; set; }
    public string Level { get; set; }
    public string ShareMethod { get; set; }
    public Lesson Lesson { get; set; }
    public int LessonId { get; set; }
    public Course Course { get; set; }
    [Required]
    public int CourseId { get; set; }
    public ICollection<Quiz> Quizs { get; set; }
  }
}
